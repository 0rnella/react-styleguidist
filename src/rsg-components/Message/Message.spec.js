import React from 'react';
import { MessageRenderer } from './MessageRenderer';

it('renderer should render message', () => {
	const message = 'Hello *world*!';
	const actual = shallow(
		<MessageRenderer classes={{}}>{message}</MessageRenderer>
	);

	expect(shallowToJson(actual)).toMatchSnapshot();
});
