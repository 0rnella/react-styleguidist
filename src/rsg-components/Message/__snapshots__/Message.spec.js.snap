exports[`test renderer should render message 1`] = `
<div>
  <_class
    text="Hello *world*!" />
</div>
`;
